subsys/bluetooth/controller/Kconfig

```diff
@@ -750,4 +750,18 @@ config BT_MAYFLY_YIELD_AFTER_CALL
 	  If set to 'n', all pending mayflies for callee are executed before
 	  yielding
 
+config BT_CTLR_USER_EXT
+	prompt "Enable proprietary extensions in Controller"
+	bool
+	help
+	  Catch-all for enabling proprietary event types in Controller behavior.
+
+config BT_CTLR_USER_EXT_RANGE
```

Comment:
maybe `BT_CTLR_USER_EVT_RANGE` ? I think the user extensions and the reserved event range can be made orthogonal perhaps?

Reply:
I agree that the ranges should be split, as they are indeed orthogonal.

Reply:
Changed the name of the constant to BT_CTLR_USER_EVT_RANGE